packages/autorest.go/src/generator/models.ts

```diff
@@ -35,19 +34,19 @@ export async function generateModels(session: Session<CodeModel>): Promise<model
   let needsJSONPopulateByteArray = false;
   let needsJSONPopulateAny = false;
   let serdeTextBody = '';
-  structs.sort((a: StructDef, b: StructDef) => { return sortAscending(a.Language.name, b.Language.name) });
+  structs.sort((a: ModelDef, b: ModelDef) => { return sortAscending(a.Language.name, b.Language.name) });
```

Comment:
Bit of a nit, should we call this local models and have this instead:
```
const models = generateModels(modelImports, serdeImports, session.model.schemas.objects);
```

If you agree please make the change in a follow up PR to avoid more noise in this one.

Reply:
Good catch, will address separately.